ophyd/device.py

```diff
@@ -805,3 +816,7 @@ def _repr_info(self):
         yield ('read_attrs', self.read_attrs)
         yield ('configuration_attrs', self.configuration_attrs)
         yield ('monitor_attrs', self.monitor_attrs)
+
+
+class RedundantStaging(Exception):
```

Comment:
Can you please move this to `ophyd.utils.errors` and subclass from `OpException`?